src/brevitas/nn/quant_mha.py

```diff
@@ -602,7 +602,8 @@ def forward(
             key_padding_mask: Optional[Tensor] = None,
             need_weights: bool = True,
             attn_mask: Optional[Tensor] = None,
-            average_attn_weights: bool = True) -> Tuple[Tensor, Optional[Tensor]]:
+            average_attn_weights: bool = True,
+            position_ids: Optional[Tensor] = None) -> Tuple[Tensor, Optional[Tensor]]:
```

Comment:
Please do not modify this forward function. `position_ids` is not in [PyTorch's MHA layer's forward function](https://pytorch.org/docs/stable/_modules/torch/nn/modules/activation.html#MultiheadAttention.forward). If this is argument is ignored, then `QuantizeOPTAttention` should handle that.